### DISCUSSION TITLE
Questions about the creation log of LSPosed

### DISCUSSION THREAD

**Discussion by Aquarius223** [LSPosed/LSPosed#1755] [Q&A] [answered]
issue: The log created by LSPosed is a bit too bloated or is there any switch to fix it?
Reason: After many users install LSPosed and boot for a period of time, the log created in "/data/adb/lspd" will increase to almost occupy the entire user space, and then mistakenly think that there is a BUG in the system (MIUI system Statistical storage space usage and other abnormal occupancy), to give me feedback on this kind of bug, I don't use LSPosed myself, so I can only ask this question here
Idea: Is it possible to add a log switch to the LSPosed settings, enable it to generate runtime work logs, and disable it to not generate log
![IMG_20220314_145007](https://user-images.githubusercontent.com/47409494/158120375-f4fd63b7-d9d3-4c02-94ab-0c8c372a3267.jpg)
s
![Screenshot_2022-03-14-14-53-55-310_com tencent mobileqq](https://user-images.githubusercontent.com/47409494/158120383-957d86bb-7c32-4a79-b806-775623eade22.jpg)

**Comment by yujincheng08** [comment, marked as answer]
It's modules that produce those logs, and we cannot control them.

From the official document:
![image](https://user-images.githubusercontent.com/5022927/158135352-729d4cb9-9b75-4bba-9f74-7f4a466ac4a3.png)
it clearly says **DON'T FLOOD THE LOG!!!**

So blame the module that floods the log.

**Reply by Aquarius223** [reply]
ok i got it, thanks for your answer
